stage1/stop/stop_fly.mk

```diff
@@ -0,0 +1 @@
+include stage1_fly/makelib/aci_binary.mk
```

Comment:
yep, it's the pid of the process: https://github.com/coreos/rkt/blob/7bb295b77ab62f6dea6e088f3042cbce43611557/stage1_fly/run/main.go#L308